frontend/components/receipts/orders/cancellation/index.tsx

```diff
@@ -1,9 +1,65 @@
+import { useState, useEffect } from 'react'
 import { ReceiptComponentProps } from '../../receipts'
 import { OrderTable } from '../../utils/order-table'
 import { ReceiptWrapper } from '../../utils/receipt-wrapper'
+import { Intent, Notification } from '@vegaprotocol/ui-toolkit'
+import { OrdersStore, useOrdersStore } from '../../../../stores/orders-store.ts'
+import { useJsonRpcClient } from '../../../../contexts/json-rpc/json-rpc-context.ts'
+
+export const locators = {
+  cancellationNotification: 'cancellation-notification',
+  cancellationView: 'cancellation-view'
+}
+
+export const CancellationNotification = ({ orderId, marketId }: { orderId: string; marketId: string }) => {
+  if (orderId) return null
+
+  return (
+    <div className="mt-2" data-testid={locators.cancellationNotification}>
+      {marketId ? (
+        <Notification intent={Intent.Warning} message={'Cancel all open orders in this market'} />
+      ) : (
+        <Notification intent={Intent.Warning} message={'Cancel all open orders in all markets'} />
+      )}
+    </div>
+  )
+}
+
+export const selectGetOrderById = (state: OrdersStore) => ({ getOrderById: state.getOrderById })
 
 export const CancellationView = ({ cancellation }: { cancellation: any }) => {
-  return <OrderTable {...cancellation} />
+  const { orderId, marketId } = cancellation
+  const [orderDetails, setOrderDetails] = useState<{ order: any; lastUpdated: number | null }>({
+    order: null,
+    lastUpdated: null
+  })
+
+  const { getOrderById } = useOrdersStore(selectGetOrderById)
```

Comment:
Once we have an error state, depending on is this goes down first or the error PR (it's lilkely going to be the error PR as it's smaller) we will need to pass that error into `RecieptView` component to ensure error states are shown appropriately.

Suggest making cancellation view presentational and moving all loading logic into the component below.